app/database/models/mentorship_relation.py

```diff
@@ -16,10 +16,10 @@ class MentorshipRelationModel(db.Model):
         action_user_id: integer indicates id of action user.
         mentor: relationship between UserModel and mentorship_relation.
         mentee: relationship between UserModel and mentorship_relation.
-        creation_date: float that defines the date of creation of the mentorship.
-        accept_date: float that indicates the date of acceptance of mentorship.
-        start_date: float that indicates the starting date of mentorship.
```

Comment:
I honestly don't understand why we use floating point numbers for dates on the mentorship system backend to being with. Should we just add this to the develop branch as well?

Reply:
I understand your point, but this is for @isabelcosta to decide 😉.